src/main/drivers/pwm_output.c

```diff
@@ -37,6 +37,18 @@ static FAST_RAM pwmCompleteWriteFn *pwmCompleteWrite = NULL;
 
 #ifdef USE_DSHOT
 FAST_RAM loadDmaBufferFn *loadDmaBuffer;
+#define DSHOT_COMMAND_DELAY_US 1000
+#define DSHOT_ESCINFO_DELAY_US 5000
+#define DSHOT_BEEP_DELAY_US 100000
+
+typedef struct dshotCommandControl_s {
```

Comment:
I suggest moving members around to avoid padding after `repeats`:
```cpp
{
  timeUs_t nextCommandAt;
  timeUs_t delayAfterCommand;
  uint8_t command[MAX_SUPPORTED_MOTORS];
  uint8_t repeats;
};

Reply:
Put repeats before command[]?  That shouldn't pad as much for the case of MAX_SUPPORTED_MOTORS is not 8? For 8, it seems like you'd get the padding either way.

Reply:
In current case padding is in the middle of the structure, which is worse than when it's at the end.

Reply:
I misread, I thought your code block was a review of the current code, not what it should be changed to.  I agree it should look exactly like that :)